src/bp/core/misc/templating.ts

```diff
@@ -17,6 +17,12 @@ export function renderRecursive(item: TemplateItem, context: any): any {
 }
 
 export function renderTemplate(template: string, context: any): string {
+  const variables = template.match(/\$[a-zA-Z]+/g) ?? []
+  for (const match of variables) {
+    const name = match.replace('$', '')
+    template = template.replace(match, `{{workflow.variables.${name}}}`)
+  }
```

Comment:
If I understand correctly this regex matches anything that looks like $abc and replaces it with the longer correct expression. So it's a shortcut to access variables?

Reply:
Yup, it's just a quick shortcut to avoid having to use the complete path to access the variable